tests/security/apparmor_profile/apache2_changehat.pm

```diff
@@ -153,13 +153,7 @@ sub run {
     my @check_list = ('file_receive', 'open', 'signal', 'mknod');
     foreach my $check_point (@check_list) {
         if ($script_output =~ m/type=AVC .*apparmor=.*DENIED.* operation=.*$check_point.* profile=.*httpd-prefork.*/sx) {
-            if (!is_sle('<=15-SP3')) {
-                record_info("ERROR", "There are denied $check_point records found in $audit_log", result => 'fail');
-                $self->result('fail');
-            }
-            else {
-                record_soft_failure('bsc#1191684 - Apparmor profile test case "apache2_changehat" found some "DENIED" audit records');
-            }
+            record_soft_failure('bsc#1191684 - Apparmor profile test case "apache2_changehat" found some "DENIED" audit records');
```

Comment:
Hi Paolo,

I would suggest you add the version check here if (!is_sle('<=15-SP4')), since o3 also has this test. https://openqa.opensuse.org/tests/2433170

Reply:
ah right, good point!